### DISCUSSION TITLE
How to preserve filename when the cursor move to nvimtree

### DISCUSSION THREAD

**Discussion by urayoru113** [nvim-tree/nvim-tree.lua#2707] [Q&A] [answered]
I'm not sure if it is possible
Is there any way to prevent `a.txt` from becoming `NvimTree_1` ?

![](https://github.com/nvim-tree/nvim-tree.lua/assets/22545315/b65036c7-7975-4e20-902d-0a978438ec93)
![](https://github.com/nvim-tree/nvim-tree.lua/assets/22545315/e4a877ec-9ec6-4449-968b-af76e9aca230)

**Comment by alex-courtis** [comment, marked as answer]
Looks like you've got a lot of customisation in place there.

Are you using a vim "distribution"? If so, I recommend asking a question there.

We can only provide help with nvim-tree itself.
